scripts/template_upload_vsphere.py

```diff
@@ -196,6 +196,8 @@ def upload_template(client, hostname, username, password,
         if not check_kwargs(**kwargs):
             return False
         if name in client.list_template():
+            logger.info("VSPHERE:%r template %r already exists", provider, name)
```

Comment:
When this was refactored a 'check_template' method was removed, I had the logic backwards here.